src/pet.js

```diff
@@ -0,0 +1,80 @@
+
+const MAXIMUM_FITNESS = 10;
+const MINIMUM_HUNGER = 0;
+const MINIMUM_NEEDFEEDING = 5;
```

Comment:
Good to see the constants grouped together here, though it seems like there are some missing. What is the maximum age? Maximum hunger? By how much are the fitness, age and hunger values incremented? By including all these in one place you avoid the need to hard code these values in the functions below, which makes for much more maintainable and easily understandable code